control-plane/agents/ha-node/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use clap::Parser;
+use grpc::operations::ha_node::{client::ClusterAgentClient, traits::ClusterAgentOperations};
+use http::Uri;
+use once_cell::sync::OnceCell;
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about, long_about = None)]
+#[clap(propagate_version = true)]
+struct Cli {
+    /// HA Cluster Agent URL or address to connect to the services.
+    #[clap(short, long, default_value = "http://ha-cluster-agent:11500")]
+    cluster_agent: Uri,
+
+    /// Node name
+    #[clap(short, long)]
+    node_name: String,
+
+    /// IP address and port for the ha node-agent to listen on
+    #[clap(short, long, default_value = "0.0.0.0:11600")]
+    grpc_endpoint: String,
+}
+
+static CLUSTER_AGENT_CLIENT: OnceCell<ClusterAgentClient> = OnceCell::new();
+
+fn cluster_agent_client<'a>() -> &'a ClusterAgentClient {
```

Comment:
You can omit <'a> lifetime and use explicitly " -> &'static ClusterAgentClient", as you always deal with static instance.